include/quill/core/Attributes.h

```diff
@@ -68,6 +68,7 @@
  */
 #ifndef QUILL_MAYBE_UNUSED
   #if QUILL_HAS_CPP_ATTRIBUTE(maybe_unused) && (defined(_HAS_CXX17) && _HAS_CXX17 == 1)
+    static_assert(false, "it's here")
```

Comment:
do we need this ? I do not understand how it is working, after this line we `#define QUILL_MAYBE_UNUSED`. Maybe you forgot to remove it ?

Reply:
I forgot to remove 🤦 
Sorry